src/WebApp/flask/app.py

```diff
@@ -239,6 +274,10 @@ def operationalization_post_operation(operation):
     operation = operation.lower()
     if operation == 'registermodel':
         model_blob_url = create_snapshot('notebooks', None, 'model.tar.gz', 'o16n')
+        if model_blob_url is None:
+            resp = Response("No serialized model found. Please train your model first before proceeding with this step. Instructions to train the model is available in the Analytics tab.", status = 400)
```

Comment:
Let's keep it concise. Don't tell them there are instructions until we actually post them.
"No serialized model found." is enough for now.